travis/test_server.py

```diff
@@ -463,6 +477,22 @@ def main(argv=None):
         return 1
     elif counted_errors != expected_errors:
         return 1
+    # no test error, let's generate .pot and msgmerge all .po files
+    must_run_makepot = (
+        os.environ.get('MQT_DISABLE_MAKEPOT') != '1' and
+        os.environ.get('TRAVIS_PULL_REQUEST') == 'false' and
+        odoo_full == 'odoo/odoo' and  # don't run makepot on OCB
```

Comment:
IMHO From OCB we should use your disable flag

Reply:
@moylop260 you mean checking the OCB environment variable?

Reply:
Change the https://github.com/OCA/OCB/blob/11.0/.travis.yml
```diff
+  - MQT_DISABLE_MAKEPOT="1"
```

Reply:
Then it's better to enable it explicity: `MAKEPOT="1"`